datacenter/dtr/2.2/guides/admin/configure/deploy-a-cache.md

```diff
@@ -140,96 +251,135 @@ In the host where you've deployed the `region-us` cache, run:
+storage:
+  delete:
+    enabled: true
+  filesystem:
+    rootdirectory: /var/lib/registry
+http:
+  addr: :5000
+  tls:
+    certificate: /certs/content-cache-ca.pem
+    key: /certs/content-cache-key.pem
 middleware:
   registry:
       - name: downstream
         options:
           blobttl: 24h
           upstreams:
-            - originhost: <Externally-reachable address for the origin registry>
-              upstreamhosts:
-                - <Externally-reachable address for upstream content cache A>
-                - <Externally-reachable address for upstream content cache B>
+            - originhost: https://<dtr-url>
           cas:
-            - <Absolute path to upstream content cache A certificate>
-            - <Absolute path to upstream content cache B certificate>
+            - /certs/dtr-ca.pem
 ```
 
-Below you can find the description for each DTR cache-specific parameter.
+And assuming that you have a certificate at `/certs/content-cache.crt` and key
+at `/certs/content-cache.key` mounted on the Content Cache container, it will
+be now serving only HTTPS requests.
 
-<table>
-  <tr>
-    <th>Parameter</th>
-    <th>Required</th>
-    <th>Description</th>
-  </tr>
-  <tr>
-    <td>
-      <code>blobttl</code>
-    </td>
-    <td>
-      no
-    </td>
-    <td>
-The TTL for blobs in the cache. This field takes a positive integer and an optional suffix indicating the unit of time. If
-this field is configured, "storage.delete.enabled" must be configured to true. Possible units are:
-      <ul>
-        <li><code>ns</code> (nanoseconds)</li>
-        <li><code>us</code> (microseconds)</li>
-        <li><code>ms</code> (milliseconds)</li>
-        <li><code>s</code> (seconds)</li>
-        <li><code>m</code> (minutes)</li>
-        <li><code>h</code> (hours)</li>
-      </ul>
-    If you omit the suffix, the system interprets the value as nanoseconds.
-    </td>
-  </tr>
-  <tr>
-    <td>
-      <code>cas</code>
-    </td>
-    <td>
-      no
-    </td>
-    <td>
-      A list of absolute paths to PEM-encoded CA certificates of upstream registries.
-    </td>
-  </tr>
-<tr>
-  <td>
-    <code>originhost</code>
-  </td>
-  <td>
-    yes
-  </td>
-  <td>
-      An externally-reachable address for the origin registry, as a fully qualified URL.
-  </td>
-</tr>
-<tr>
-  <td>
-    <code>upstreamhosts</code>
-  </td>
-  <td>
-    no
-  </td>
-  <td>
-    A list of externally-reachable addresses for upstream registries for cache chaining. If more than one host is specified, pulls from upstream content caches will be done in round-robin order.
-  </td>
-</tr>
-</table>
+### Let's Encrypt
+
+Content Cache also supports using Let's Encrypt to automatically obtain a
+browser-trusted certificate.  For more information on Let's Encrypt, see
+[https://letsencrypt.org/how-it-works/](https://letsencrypt.org/how-it-works/)
+and the relevant section of the [registry configuration](/registry/configuration.md#letsencrypt).
+
+### Alternatives
```

Comment:
Yup. We need to improve the distribution docs to remove ambiguities like that. Don't use those docs as a reference of the quality we should be aiming for.